src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -752,7 +753,10 @@ public static void GetFullPath_Windows_83Paths()
                     string shortName = sb.ToString();
 
                     // Make sure the shortened name expands back to the original one
-                    Assert.Equal(tempFilePath, Path.GetFullPath(shortName));
+                    // Sometimes shortening or GetFullPath is changing the casing of "temp" on some test machines: normalize both sides
+                    tempFilePath = Regex.Replace(tempFilePath, @"\temp\\", @"\TEMP\\",  RegexOptions.IgnoreCase);
```

Comment:
Wouldn't a string replace be sufficient? (faster than regex) I think the IgnoreCase parameter isn't really needed here. But I guess we aren't caring about performance in tests a lot.